packages/react-components/src/organisms/__tests__/TeamCreateOutputForm.test.tsx

```diff
@@ -165,11 +162,12 @@ describe('on submit', () => {
     });
   };
   const submitForm = async () => {
-    const button = screen.getByRole('button', { name: /Share/i });
-    userEvent.click(button);
+    userEvent.click(screen.getByRole('button', { name: /Share/i }));
 
-    expect(screen.getByRole('button', { name: /Share/i })).not.toBeEnabled();
-    expect(screen.getByRole('button', { name: /Cancel/i })).not.toBeEnabled();
+    waitFor(() => {
```

Comment:
I belive this returns a promise so without an away I am not sure this works as intended.